review10/passenger-service/src/main/java/cz/muni/fi/airportmanager/passengerservice/service/PassengerService.java

```diff
@@ -24,6 +26,12 @@ public class PassengerService {
     @RestClient
     BaggageClientResource baggageClientResource;
 
+    @ConfigProperty(name = "baggage-service.rest.username")
+    String username;
+
+    @ConfigProperty(name = "baggage-service.rest.password")
+    String password;
```

Comment:
Do you need to have these here when they are only used in the client header class?